internal/cli/atlas/quickstart/dbuser_setup.go

```diff
@@ -84,8 +82,8 @@ func (opts *Opts) validateUniqueUsername(val interface{}) error {
 
 	_, err := opts.store.DatabaseUser(convert.AdminDB, opts.ConfigProjectID(), username)
 	if err != nil {
-		var target *atlas.ErrorResponse
-		if errors.As(err, &target) && target.ErrorCode == "USERNAME_NOT_FOUND" {
+		target, ok := atlasv2.AsError(err)
+		if ok && target.GetErrorCode() == "USERNAME_NOT_FOUND" {
```

Comment:
Same ad above. I targeted to use one helper for situations when we need error code?
Maybe we should also return error object along with boolean?

Reply:
we can anyway access the values using `target.Get...` methods